app/views/coronavirus_pages/_timeline_entries.html.erb

```diff
@@ -0,0 +1,30 @@
+<% timeline_entries =
+ @coronavirus_page.timeline_entries.order(:position).map do |timeline_entry|
+   {
+     id: timeline_entry.id,
+     field: timeline_entry.heading,
+     edit: { href: coronavirus_page_path(@coronavirus_page.slug) },
```

Comment:
The "Edit timeline page" page exists now, so this could link to `edit_coronavirus_page_timeline_entry(@coronavirus_page.slug, timeline_entry)`